Makefile

```diff
@@ -1,8 +1,22 @@
 #
 # Makefile for more convenient building of the Linode CLI and its baked content
 #
+PKG_PATH =
+TEST_SUITE_PATH =
+TEST_CASE_COMMAND =
```

Comment:
Can we have some explicit default values for these vars?

Reply:
Maybe some empty strings, `""`?

Reply:
It is best left null for these variables because `testint` command will actually run the entire test suite when left empty. However if we give some empty string value such as "", pytest will not function properly.
For reference, 	`pytest tests/integration$(PKG_PATH)$(TEST_SUITE_PATH) $(TEST_CASE_COMMAND)
`